ReactAndroid/src/main/jni/CMakeLists.txt

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+cmake_minimum_required(VERSION 3.13)
+set(CMAKE_VERBOSE_MAKEFILE on)
+
+project(ReactAndroid)
+
+# Make sure every shared lib includes a .note.gnu.build-id header
+add_link_options(-Wl,--build-id)
```

Comment:
is there any specific reason for this? this [breaks reproducibility](https://gitlab.com/IzzyOnDroid/repo/-/wikis/Reproducible-Builds/RB-Hints-for-Developers#no-funny-build-time-generated-ids).